### DISCUSSION TITLE
ERROR: Couldn't find env file: /home/malith/hosting.properties

### DISCUSSION THREAD

**Discussion by LiyanageMX** [Budibase/budibase#2297] [Help] [answered]
docker-compose --env-file hosting.properties up
when i using this command it shows this error.
why this happen.Plz help me

**Comment by mjashanks** [comment, marked as answer]
It looks like your hosting.properties file does not exist in that folder.

Maybe you should try using the CLI instead: https://docs.budibase.com/getting-started/budibase-cli-setup

**Reply by LiyanageMX** [reply]
i downloaded those three files.but i dont know where to save the.Is there seperate folder to save them?

**Reply by mjashanks** [reply]
Create a new directory, anywhere at all. Put those three files inside. Then, `cd` into that directory and run the `docker-compose` command, as before.

**Reply by LiyanageMX** [reply]
thank you
